server/src/main/java/org/opensearch/bootstrap/SystemCallFilter.java

```diff
@@ -260,6 +260,7 @@ static class Arch {
         m.put("amd64", new Arch(0xC000003E, 0x3FFFFFFF, 57, 58, 59, 322, 317));
         m.put("aarch64", new Arch(0xC00000B7, 0xFFFFFFFF, 1079, 1071, 221, 281, 277));
         m.put("s390x", new Arch(0x80000016, 0xFFFFFFFF, 2, 190, 11, 354, 348));
+        m.put("ppc64le", new Arch(0xC0000015, 0xFFFFFFFF, 2, 189, 11, 362, 358));
```

Comment:
(Preference) I would really like if at least some these values could be made constants.   I see above they correspond to [`AUDIT_ARCH_XXX`](AUDIT_ARCH_XXX), [`__NR_xxx`](https://github.com/torvalds/linux/blob/479174d402bcf60789106eedc4def3957c060bad/tools/testing/selftests/seccomp/seccomp_bpf.c#L130), etc.